src/main/scala/com/typesafe/sbt/pom/PomReaderPlugin.scala

```diff
@@ -0,0 +1,15 @@
+package com.typesafe.sbt.pom
+
+import sbt._
+import sbt.plugins.JvmPlugin
+
+/** Plugin definition for Maven POM reader. */
+object PomReaderPlugin extends AutoPlugin {
+  object autoImport extends SbtPomKeys {
+    @deprecated("This is an AutoPlugin; settings are automatically added. Using this value is no longer required", "1.1")
+    lazy val useMavenPom = projectSettings
+  }
+  override def requires = JvmPlugin
+  override def trigger = allRequirements
```

Comment:
I waffled on this one. In the end, I leaned toward less work for the users, given that in most cases with this plugin, the `pom.xml` file is the authoritative build definition, and the user is likely to just want to get something that loads into sbt with minimal ceremony. That said, it's your call :-)